recipes/poselib/all/conanfile.py

```diff
@@ -76,15 +76,16 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.cache_variables["MARCH_NATIVE"] = False
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
         tc.generate()
 
         deps = CMakeDeps(self)
         deps.generate()
 
     def _patch_sources(self):
-        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
-                        "-march=native -Wall -Werror -fPIC", "-Wall")
+        cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
+        replace_in_file(self, cmakelists, "-Werror -fPIC", "")
```

Comment:
```suggestion
        replace_in_file(self, cmakelists, "-march=native -Wall -Werror -fPIC", "")
```
Revert to the previous from in version 2.0.3